### ISSUE TITLE
Empty initialize method produces a Lint/DisjunctiveAssignmentInConstructor error

### ISSUE THREAD

**Issue by Zajn** [rubocop-hq/rubocop#6681]
I'm unsure if this is intended behavior or not, but the error message told me to submit an issue.

## Expected behavior
No errors are output to the terminal. 

## Actual behavior
This error was produced by Rubocop:

```
An error occurred while Lint/DisjunctiveAssignmentInConstructor cop was inspecting /Users/zach/workspace/project/lib/sql_function/capitalizer_map.rb:5:4.
```

## Steps to reproduce the problem

The smallest file that reproduces this problem is this:

```ruby
# frozen_string_literal: true

# top-level class documentation comment
class Test
  def initialize; end
end
```

Running `rubocop` in a directory that only contains this file results in:

```
Inspecting 1 file
An error occurred while Lint/DisjunctiveAssignmentInConstructor cop was inspecting /Users/zach/workspace/rubocop-test/test.rb:5:2.
To see the complete backtrace run rubocop -d.
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while Lint/DisjunctiveAssignmentInConstructor cop was inspecting /Users/zach/workspace/rubocop-test/test.rb:5:2.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
```
The full backtrace from `rubocop -d` is here:

```
~/workspace/rubocop-test » rubocop -d                                                                                                                                    2.5.3 
For /Users/zach/workspace/rubocop-test: configuration from /Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/config/default.yml
Inspecting 1 file
Scanning /Users/zach/workspace/rubocop-test/test.rb
An error occurred while Lint/DisjunctiveAssignmentInConstructor cop was inspecting /Users/zach/workspace/rubocop-test/test.rb:5:2.
undefined method `type' for nil:NilClass
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/lint/disjunctive_assignment_in_constructor.rb:43:in `check_body'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/lint/disjunctive_assignment_in_constructor.rb:37:in `check'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/lint/disjunctive_assignment_in_constructor.rb:29:in `on_def'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/commissioner.rb:58:in `block (2 levels) in trigger_responding_cops'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/commissioner.rb:106:in `with_cop_error_handling'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/commissioner.rb:57:in `block in trigger_responding_cops'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/commissioner.rb:56:in `each'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/commissioner.rb:56:in `trigger_responding_cops'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/commissioner.rb:34:in `block (2 levels) in <class:Commissioner>'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/ast/traversal.rb:92:in `on_class'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/commissioner.rb:35:in `block (2 levels) in <class:Commissioner>'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/ast/traversal.rb:13:in `walk'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/commissioner.rb:46:in `investigate'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/team.rb:116:in `investigate'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/team.rb:104:in `offenses'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cop/team.rb:44:in `inspect_file'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:280:in `inspect_file'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:227:in `block in do_inspection_loop'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:259:in `block in iterate_until_no_changes'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:252:in `loop'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:252:in `iterate_until_no_changes'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:223:in `do_inspection_loop'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:126:in `block in file_offenses'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:144:in `file_offense_cache'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:124:in `file_offenses'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:112:in `process_file'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:89:in `block in each_inspected_file'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:86:in `each'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:86:in `reduce'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:86:in `each_inspected_file'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:76:in `inspect_files'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/runner.rb:48:in `run'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cli.rb:174:in `execute_runner'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cli.rb:75:in `execute_runners'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/lib/rubocop/cli.rb:47:in `run'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/exe/rubocop:13:in `block in <top (required)>'
/Users/zach/.rubies/ruby-2.5.3/lib/ruby/2.5.0/benchmark.rb:308:in `realtime'
/Users/zach/.gem/ruby/2.5.3/gems/rubocop-0.63.0/exe/rubocop:12:in `<top (required)>'
/Users/zach/.gem/ruby/2.5.3/bin/rubocop:23:in `load'
/Users/zach/.gem/ruby/2.5.3/bin/rubocop:23:in `<main>'
```

## RuboCop version

```
$ bundle exec rubocop -V
0.63.0 (using Parser 2.6.0.0, running on ruby 2.5.3 x86_64-darwin18)
```

**Comment by Drenmi** [comment]
Thanks for the report! This was reported in #6677 and #6678, and fixed in #6679. The error will go away in the next release. 🙇
